test/python/common.py

```diff
@@ -324,12 +324,16 @@ def _(*args, **kwargs):
             # Special case: instead of using the standard credentials mechanism,
             # load them from different environment variables. This assumes they
             # will always be in place, as is used by the Travis setup.
+            _hub = os.getenv('IBMQ_HUB')
+            _group = os.getenv('IBMQ_GROUP')
+            _project = os.getenv('IBMQ_PROJECT')
+            _url = "https://q-console-api.mybluemix.net/api/" + \
+                "Hubs/{hub}/Groups/{group}/Projects/{project}"
+            _url = _url.format(hub=_hub, group=_group, project=_project)
+
             kwargs.update({
                 'QE_TOKEN': os.getenv('IBMQ_TOKEN'),
-                'QE_URL': os.getenv('IBMQ_URL'),
-                'hub': os.getenv('IBMQ_HUB'),
-                'group': os.getenv('IBMQ_GROUP'),
-                'project': os.getenv('IBMQ_PROJECT'),
+                'QE_URL': _url
```

Comment:
Can you actually remove the lines below (327-333) and just leave this as:
```
kwargs.update({
    'QE_TOKEN': os.getenv('IBMQ_TOKEN'),
    'QE_URL': os.getenv('IBMQ_URL')
})
```
We basically have full control over those variables as it is exclusively used for the "travis jobs for premium" use case, and it is just easier to update the environment variable directly (I'll do it right before merging the PR).